frontend/src/views/Administration.vue

```diff
@@ -149,6 +149,48 @@ SPDX-License-Identifier: Apache-2.0
                     </v-list-item-content>
                   </v-list-item>
                 </template>
+                <template v-if="customFieldsListShoot">
```

Comment:
The name should be `customShootFieldList` or `customShootFields`

Reply:
will rename to `shootCustomFields` and `shootCustomFieldList` respectively

Reply:
✅